XcodeProjAdder/xprojZoneFileTools.cpp

```diff
@@ -92,5 +92,10 @@ bool ZoneFileTools::isFileTypeSupported(std::string ext)
 
 bool ZoneFileTools::in_array(const string &value, const vector<string> &array)
 {
-    return std::find(array.begin(), array.end(), value) != array.end();
```

Comment:
I can't say I fully understand why both clang and gcc trip over on that code, but here's the error, as provided by `clang++`:
```
xprojZoneFileTools.cpp:95:12: error: no matching function for call to 'find'
    return std::find(array.begin(), array.end(), value) != array.end();
           ^~~~~~~~~
/usr/bin/../lib64/gcc/x86_64-unknown-linux-gnu/5.3.0/../../../../include/c++/5.3.0/bits/streambuf_iterator.h:369:5: note: candidate template ignored: could not match 'istreambuf_iterator' against '__normal_iterator'
    find(istreambuf_iterator<_CharT> __first,
    ^
1 error generated.
```